tests/test_black.py

```diff
@@ -1389,6 +1389,32 @@ def test_include_exclude(self) -> None:
         )
         self.assertEqual(sorted(expected), sorted(sources))
 
+    def test_gitingore_used_as_default(self) -> None:
```

Comment:
Is it worth it to add another test that's like this one but the complete opposite? So like making sure .gitignore is not used when --exclude is present. OR is that already tested somewhere else (wouldn't surprised if so)?

Reply:
All of the existing tests test this already since they all pass `exclude`. This is the only one that doesn't.